### PR TITLE
VACMS-2430: Remove seconds input from datetime fields.

### PR DESCRIPTION
**Pull request by ethanteague**
## Description
See #2430 

## Testing done
Front end build / visual

## Screenshots
image

## QA steps
- As an admin, go to `node/6920/edit` and visually verify that seconds input does not appear in date widget.
- Change time, ensure node saves without errors.

### CODE REVIEW

File: docroot/modules/custom/va_gov_backend/va_gov_backend.module [hunk 1]
```diff
@@ -38,6 +38,24 @@ function va_gov_backend_help($route_name, RouteMatchInterface $route_match) {
   }
 }
 
+/**
+ * Implements hook_element_info_alter().
+ */
+function va_gov_backend_element_info_alter(array &$types) {
+  $types['datetime']['#process'][] = 'va_gov_backend_datetime_set_format';
```
Comment (swirtSJW) [inline docroot/modules/custom/va_gov_backend/va_gov_backend.module:45]: I don't think we want to add this blindly.   I think we only want to set this if $types['datetime'] is set.

Comment (swirtSJW) [reply docroot/modules/custom/va_gov_backend/va_gov_backend.module:45]: Also `$type` is D7,  `$info` is D8   and `$types` is Dnothin.

**Comment by swirtSJW** [comment]
Has some conflicts that need resolving.

**Comment by ethanteague** [comment]
> Has some conflicts that need resolving.

Updated, @swirtSJW :+1:

**Pull request merged**